apps/web/src/app/layout.tsx

```diff
@@ -37,6 +41,18 @@ export default async function RootLayout({
   const shouldInjectToolbar = process.env.NODE_ENV === 'development';
   return (
     <Html className={clsx(lato.variable, sourceSans.variable)}>
+      <head>
+        <link
+          rel="icon"
+          href="https://hypha.earth/wp-content/uploads/2023/07/cropped-favicon-32x32.png"
+          sizes="32x32"
+        />
+        <link
+          rel="icon"
+          href="https://hypha.earth/wp-content/uploads/2023/07/cropped-favicon-192x192.png"
+          sizes="192x192"
+        />
+      </head>
```

Comment:
This is duplicate now. I've moved that to `Html` component in my previous layout PR which you've reviewed 😉

Reply:
```suggestion
```